content/blog/deploying-super-mario-game-on-kubernetes-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,477 @@
+---
+title: Deploying Super Mario game on Kubernetes in HPE GreenLake for Private
+  Cloud Enterprise
+date: 2024-03-29T17:53:09.850Z
+author: Guoping Jia
+authorimage: /img/guoping.png
+disable: false
+tags:
+  - HPE GreenLake for Private Cloud Enterprise
+  - Load balancer
+  - SSL/TLS certificates
+  - Ingress
+  - Ingress controller
+  - Super Mario
+  - Tetris
+  - hpe-greenlake-for-private-cloud-enterprise
+  - TLS termination
+---
+<style> li { font-size: 27px; line-height: 33px; max-width: none; } </style>
+
+Continuing on from the the blog post [Exposing applications using Ingress and TLS termination on Kubernetes in HPE GreenLake for Private Cloud Enterprise](https://developer.hpe.com/blog/exposing-an-application-using-ingress-and-tls-termination-on-kubernetes-in-hpe-greenlake-for-private-cloud-enterprise/), I will walk you through the steps of deploying real-world game applications such as the legendary *Super Mario* and *Tetris* on Kubernetes (K8s) in the HPE GreenLake for Private Cloud Enterprise. By utilizing K8s Ingress, TLS termination, and a range of suitable tools, *Super Mario*, together with the game *Tetris*, is available and securely accessible via HTTPS. This setup strictly adheres to the rigorous security and compliance standards of the K8s production environment in HPE GreenLake for Private Cloud Enterprise.
```

Comment:
Guoping,
We have an opportunity here to really engage the reader. What you wrote is perfectly fine and grammatically correct. But consider having a little more fun with it. I'll write up a little something below that might help you brainstorm a bit on it:

In my recent blog post, I showed you how to expose applications using Ingress and TLS termination on Kubernetes in HPE GreenLake for Private Cloud Enterprise. [you can link to that blog post here... the words don't have to totally match the title.] Let's have a little fun practicing this through a real-world use case where I walk you through the steps of deploying gaming applications, like Super Mario and Tetris on Kubernetes (K8s) in HPE GreenLake for Private Cloud Enterprise. By using K8s Ingress, TLS termination, and a range of tools, Super Mario and Tetris can be made available and securely accessible via HTTPS. The setup I show here strictly adheres to the rigorous security and compliance standards of the K8s production environment in HPE GreenLake for Private Cloud Enterprise.

Reply:
Thanks Dale ! This is a very good suggestion. 

I've updated it with what you wrote up. Please review.